src/main/scala/com/fulcrumgenomics/commons/reflect/ReflectionUtil.scala

```diff
@@ -387,7 +387,7 @@ object ReflectionUtil {
     }
     else if (clazz == classOf[Option[_]]) {
       // unitType shouldn't be an option, since it's supposed to the type *inside* any containers
-      throw new ReflectionException(s"Cannot construct an '${unitType.getSimpleName}' from a string.  Is this an Option[Option[...]]?")
+      throw new ReflectionException(s"Is this an Option[Option[...]]? Cannot construct an '${unitType.getSimpleName}'from a string: $value")
```

Comment:
I added this in, since I want to know the string value too to help me figure out which one it was!